src/hotspot/share/gc/shenandoah/shenandoahConcurrentGC.cpp

```diff
@@ -914,7 +907,7 @@ void ShenandoahConcurrentGC::op_cleanup_early() {
 }
 
 void ShenandoahConcurrentGC::op_rendezvous_roots() {
```

Comment:
This method is no longer needed, so as disable_concurrent_weak_root_in_progress_concurrently()